common/include/common/stringutils.h

```diff
@@ -28,4 +30,7 @@ int valloc_sprintf(char ** str, const char * format, va_list ap)
 int alloc_sprintf(char ** str, const char * format, ...)
   __attribute__ ((format (printf, 2, 3)));
 
+// Find value in a list separated by delimiter.
+bool util_containsValue(const char * list, char delimiter, const char * value);
```

Comment:
Sorry, but one last thing, this is no longer part of the `util_` function set, perhaps `str_`?

Reply:
Done.